### ISSUE TITLE
Feature Requests

### ISSUE THREAD

**Issue by TyPilot** [Assaro/DD_Bot#8]
All requests are low priority and are things that "would be nice". 

1. Show how many players are online when using the `/list` command
2. Split Permissions to per docker container for cross-functional server bot use. Meaning, specify a user ID per docker container for a user to have access to. 
3. Make command output prettier. (See image below)

Bonus because I realize this one could be way to much work for little payout
4. Ability to grant users access with a new command. Will need admin writes for that user in order for these commands to work. Add a new command called `/grant user:USERNAME server:CONTAINER_NAME`. Could use user ID instead of username depending on the complexity. This would help with permissions without having to edit files or if another admin doesn't have access to the edit the files or doesn't know how. 

<img width="541" alt="discord-gsm" src="https://user-images.githubusercontent.com/7003699/196024293-73115015-b14c-4a4b-bf6f-3edb4e57dfe5.png">

**Comment by Assaro** [comment]
Some of those came to my mind already, here is what i will probably do:

1.  Showing the amount of players at the /list command is not really possible. You cannot get those infos from docker itself. You would have to implement that for every possible program specifically. If you do it with RCON, which most gameservers support, you need a whole section of settings just for that, each having password, ip, port and probably rcon protocol. Not in Scope. Won't do. Maybe as an Add-On in the far future
2. Split permissions can and probably will be done. However a settings rework is needed first
3. Are you talking about updates to the bot or updates to other docker containers?
4. I could make it a little prettier, i will however keep using that list for it as i want to have the ability to glance over it quickly, not have it take up the whole screen. A `/details container:ContainerName` might be an idea though.
5. If i implement Your second idea, this will be definitely done. I can however only do it if i rewrite my json implementation. Since i am working on that anyway, i will look into it

**Comment by TyPilot** [comment]
3. Updates for the docker containers

![Screenshot_20221016_110625_Firefox](https://user-images.githubusercontent.com/7003699/196045816-27494476-5ea2-4d82-a316-272ac6a43505.jpg)

**Comment by Assaro** [comment]
The check for updates is not a docker feature
It is entirely an unraid thing. I don't know if i will be able to work with that. I will have a look into it though

**Comment by chris3081** [comment]
It might be easier for the requestor to use tools like WatchTower https://github.com/containrrr/watchtower

**Comment by Assaro** [comment]
> It might be easier for the requestor to use tools like WatchTower https://github.com/containrrr/watchtower 

Agreed. This is definitely a better solution than whatever i could come up with.
